src/algorithm/joint-configuration.hpp

```diff
@@ -303,16 +315,14 @@ namespace se3
     template<typename JointModel>
     static void algo(const se3::JointModelBase<JointModel> & jmodel,
                      se3::JointDataBase<typename JointModel::JointData> &,
-                     const se3::Model & model,
+                     const se3::Model & ,
```

Comment:
jmodel and jdata cannot be removed as it is needed by JOINT_VISITOR_INIT.
model and data will be removed when not useful